tests/prepare-docker-nextcloud-container.ts

```diff
@@ -22,7 +22,7 @@ import configuration   from "./configuration";
     times += 1;
 
     if (times > 10) {
-      console.log("The nextcloud container does not seem to work. Aborting…");
+      console.log('The nextcloud container does not seem to work. Aborting…');
```

Comment:
container? It's not always a container.

Reply:
I think this might have been copied from the WebDav-client module. 
All I did was change the quotation marks.
What would you like it to say?

Reply:
Changing the quotation mark in this PR is a bad idea IMO. It makes reading it unnecessarily difficult. I realize it is kinda late and you have other fish to fry, but make sure to do a separate PR next time for style changes. You probably don't even need a PR for that kind of things :)

Reply:
I will make sure to do so in the future ^^'